### PR TITLE
Release mod-oai-pmh 1.2.1

### PR DESCRIPTION
**Pull request by javalkar-varun**
**Purpose:**
https://issues.folio.org/browse/MODOAIPMH-78 - release 1.2.1

**Approach:**
- Update NEWS.md
- Update interface dependencies

### CODE REVIEW

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -76,7 +76,7 @@
     },
     {
       "id": "source-storage-records",
-      "version": "1.0"
+      "version": "3.0"
```
Comment (pautau-s) [inline descriptors/ModuleDescriptor-template.json]: mod-source-storage-records has not increased source-storage-records interface version. Please leave it unchanged

Comment (javalkar-varun) [reply descriptors/ModuleDescriptor-template.json]: It has increased https://github.com/folio-org/mod-source-record-storage/releases/tag/v3.0.0. So why not update it?

Comment (pautau-s) [reply descriptors/ModuleDescriptor-template.json]: 3.0.0 is a module version not interface

File: NEWS.md [hunk 2]
```diff
@@ -1,4 +1,12 @@
-## 1.2.1 (UnReleased)
+## 1.2.2 (UnReleased)
+
+## 1.2.1 (Released)
+This release was to tune and improve environment settings. Also, update interface dependencies to avoid breaking mod-oai-pmh.
```
Comment (pautau-s) [inline NEWS.md]: Remove please second sentence since there is no interface update

File: NEWS.md [hunk 3]
```diff
@@ -1,4 +1,12 @@
...
+
+[Full Changelog](https://github.com/folio-org/mod-oai-pmh/compare/v1.2.0...v1.2.1)
+
+### Stories
+* [FOLIO-2358](https://issues.folio.org/browse/FOLIO-2358) Use JVM features to manage container memory
```
Comment (pautau-s) [inline NEWS.md]: [MODOAIPMH-80](https://issues.folio.org/browse/MODOAIPMH-80)  there is a ticket in MODAIPMH project. Please use it instead of FOLIO-2358

**Pull request merged**